package.json

```diff
@@ -34,7 +34,8 @@
   },
   "peerDependencies": {
     "react": "^16.14.0",
-    "react-dom": "^16.14.0"
+    "react-dom": "^16.14.0",
+    "react-router-dom": "^6.3.0"
```

Comment:
yeah, now we have to depend on one more thing, but `Link` doesn't work otherwise

considering maybe downgrading the peer version, as the `Link` API hasn't changed much since much older versions